bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/AsterismType.java

```diff
@@ -2,38 +2,58 @@
 
 import edu.gemini.pot.sp.SPComponentBroadType;
 import edu.gemini.pot.sp.SPComponentType;
-import edu.gemini.shared.util.immutable.DefaultImList;
-import edu.gemini.shared.util.immutable.ImList;
+import edu.gemini.shared.util.immutable.*;
+import edu.gemini.spModel.gemini.ghost.AsterismConverters;
 import edu.gemini.spModel.gemini.ghost.GhostAsterism$;
 
+import java.util.function.Supplier;
+
 public enum AsterismType {
-    Single("single") {
-        @Override
-        public Asterism createEmptyAsterism() {
-            return Asterism$.MODULE$.createSingleAsterism();
-        }
-    },
-    GhostStandardResolution("ghostStandardRes") {
-        @Override
-        public Asterism createEmptyAsterism() {
-            return GhostAsterism$.MODULE$.createEmptyStandardResolutionAsterism();
-        }
-    },
-    GhostHighResolution("ghostHighRes") {
-        @Override
-        public Asterism createEmptyAsterism() {
-            return GhostAsterism$.MODULE$.createEmptyHighResolutionAsterism();
-        }
-    },
+    /**
+     * Note all GHOST standard resolution asterisms have the same tag because we use this to output them to XML
+     * in a category together.
+     */
```

Comment:
Hmm, I wonder if it would have been better to make different actual `Asterism` instances for all the different standard res options instead of having them all grouped under standard resolution.  Then you wouldn't have two levels of stuff and `AsterismType` would be 1:1 with `Asterism` instances.  That seems cleaner to me now.

Reply:
Yeah, I agree that the extra layer adds a certain amount of possibly unnecessary complexity.

I guess we could consider making `StandardResolution` a `sealed trait` or `abstract sealed class`, and then make four `case class`es off of that, one for each asterism type?